modules/govuk_postgresql/manifests/backup.pp

```diff
@@ -34,4 +34,7 @@
         source => 'puppet:///modules/govuk_postgresql/etc/postgresql-backup-pre',
     }
 
+    # Include offsite backups to S3
+    include wal_e::backup
```

Comment:
I think I'd find this a bit surprising if I was looking back at this code later. I'd look for this class in an external `wal_e` module or in a directory called `modules/wal_e/` rather than `modules/govuk_postgresql/manifests/wal_e/`. Is that just me?